evals/elsuite/text_compression/requirements.txt

```diff
@@ -0,0 +1,2 @@
+jiwer==3.0.2
```

Comment:
I think these need to be added to https://github.com/openai/evals/blob/main/pyproject.toml